scrunch/expressions.py

```diff
@@ -526,7 +526,7 @@ def category_ids(var_id, var_value, variables=variables):
         if len(subitems) == 2:
             if 'value' in subitems[1] and 'variable' in subitems[0]:
                 var_url = subitems[0]['variable']
-                if ds.variables.index[var_url]['type'] == 'multiple_response':
+                if var_url in ds.variables.index and ds.variables.index[var_url]['type'] == 'multiple_response':
```

Comment:
Careful with doing this. You probably fetched the catalog index before. This is doing 2 independent GET requests to /dataset/id/variables/ in the same line. It is better to first fetch `ds.variables.index` in one variable and then do these checks. Or even better, here https://github.com/Crunch-io/scrunch/blob/46737cd15f550c76a033844813eeee6652ff99f4/scrunch/expressions.py#L474 you fetched all the variables, probably that has the information you need for this condition without making this request.

I am ware of this request because the datasets that are being worked on can have around hundred thousands variables, so this catalog is not cheap to obtain.